src/app/lib/helpers.ts

```diff
@@ -48,3 +48,5 @@ export function concat(...parts: Uint8Array[]) {
   }
   return result
 }
+
+export const parseNumberToBigInt = (value: number) => BigInt(Math.round(value * 10 ** 9))
```

Comment:
@pro-wh is this usually called converting to base units?

I'm not sure if that's accurate for `parseNumberToBigInt(action.payload.shares)`

Reply:
Should it be something else? Lots of software, e.g. this and oasis scan, seem to be using giga shares.

Reply:
Not sure.

I rebased. Happy to merge

Reply:
ok let's proceed to merge. there's been some talk of moving away from presenting shares or gigashares to end users completely, so we'll have a lot more design discussion when we do that